README.md

```diff
@@ -1,44 +1,35 @@
-# Pngme Android (React Native) SDK & Sample App
+<p align="center">
+  <img src="https://admin.pngme.com/logo.png" alt="Pngme" width="100" height="100">
+  <h3 align="center">Pngme Android (React Native) SDK & Sample App</h3>
+</p>
 
-*Welcome to the Pngme v2.x React Native SDK!*<br>
-This Readme will cover how the SDK works, get-started basics, and a sample Android app.
+This documentation covers how to use the Kotlin SDK v2.x with React Native.
```

Comment:
Is referencing Kotlin here confusing?
For RN, we do have an npm package (https://www.npmjs.com/package/@pngme/react-native-sms-pngme-android).
The npm package is a thin wrapper around our core Kotlin sdk assets... but a RN developer doesn't need to know or understand that. A dev should be able to do a pure RN integration (unlike Flutter, for which the current integration requires touching Kotlin and managing some cross-compile complexities).

Reply:
Good point, copied from the flutter app docs. Removed.